<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select_s4</class>
 <widget class="QDialog" name="select_s4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_category">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_item">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>541</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>AvgRate</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit_avgRate">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
